.eleventy.js

```diff
@@ -41,8 +41,10 @@ module.exports = function (eleventyConfig) {
   // Good candidate for 11ty-build-system.
   eleventyConfig.addFilter("changeDomain", function (url, domain) {
     try {
-      let u = new URL(url, `https://${domain}`);
+      let host = config.build.canonical_url.split("//");
```

Comment:
I'm guessing these changes are necessary because you're serving some versions of this site over HTTP (not HTTPS)? 

If so, I wonder if there's a cleaner way to do this by just including the http/https protocol in the `DOMAIN` environment variable. That would give us really specific control for each build in whichever GitHub Action does the build, etc. The main motivation behind this `changeDomain` filter was to ensure it'd work with all the different PR preview subdomains in Drought, so I'm all about extending that power to other use cases.

At any rate, not a blocker, just an idea.

Reply:
Yeah, I agree - was thinking of messaging you about it. 
We can make an issue for it in odi-engineering & assign it to the static site board. I'll add that to my to-do list unless you want to do it & record your thoughts.

I was avoiding adding this in the config file - that seems excessive. Would be good to understand what we want to grow into and what we need to support.